src/components/IdeaCard.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+import Draggable from "react-draggable";
+
+import { useDispatch } from "react-redux";
```

Comment:
importing `react-redux` twice. I love the order and i see you're trying to make a sense of it so I'll tell you what i use:

1. Libraries (all external packages, react, react-redux, react-router, express, axios, etc)
2. My files (actions/selectors in the FE, routers models middlewares. in the BE)
3. Components
4. Css

😄